pkg/kptfile/pkgfile.go

```diff
@@ -48,10 +48,7 @@ type KptFile struct {
 	Dependencies []Dependency `yaml:"dependencies,omitempty"`
 
 	// OpenAPI contains additional schema for the resources in this package
-	// Uses interface{} instead of Node to work around yaml serialization issues
-	// See https://github.com/go-yaml/yaml/issues/518 and
-	// https://github.com/go-yaml/yaml/issues/575
-	OpenAPI interface{} `yaml:"openAPI,omitempty"`
+	OpenAPI *yaml.Node `yaml:"openAPI,omitempty"`
```

Comment:
So this feels a bit strange. While we always serialize this data structure to yaml, the data structure itself isn't really tied to yaml.

Reply:
What do you mean by it isn't tied to yaml ?